src/cpp/session/modules/SessionFind.cpp

```diff
@@ -1095,6 +1127,23 @@ struct ReplaceOptions
    const std::string replacePattern;
 };
 
+void addPackageDirectoriesToCommand(const FilePath& directoryPath, shell_utils::ShellCommand* pCmd)
+{
+   FilePath rPath(string_utils::utf8ToSystem(directoryPath.getAbsolutePath() + "/R"));
+   FilePath srcPath(string_utils::utf8ToSystem(directoryPath.getAbsolutePath() + "/src"));
+   FilePath testsPath(string_utils::utf8ToSystem(directoryPath.getAbsolutePath() + "/tests"));
+
+   *pCmd << shell_utils::EscapeFilesOnly << "--" << shell_utils::EscapeAll;
```

Comment:
To be honest this is pulled from the existing code, but my understanding is that it exists to prevent the `--` from being escaped. There is no EscapeNone option but I can add a comment.